flow/project.py

```diff
@@ -1720,9 +1717,14 @@ def _run_operations_in_parallel(self, pool, pickle, operations, progress, timeou
         for result in tqdm(results) if progress else results:
             result.get(timeout=timeout)
 
-    def _execute_operation(self, operation, timeout=None):
-        logger.info("Execute operation '{}'...".format(operation))
+    def _execute_operation(self, operation, timeout=None, pretend=False):
+        if not pretend:
```

Comment:
Small thing here. We don't need to do the check if this goes after the `if pretend:` condition. Please move this beneath that conditional. Since we return None if `pretend is True`, we don't need to use an else.

Reply:
Sure thing :)